src/hooks/useUserLocation.js

```diff
@@ -11,13 +11,16 @@ export default function useUserLocation() {
     })
     const [countyData, setCountyData] = useState({
         countyName: '',
-        incidence: 0
+        incidence: 0,
+        last_update: ''
     })
     const [ errorMessage, setErrorMessage ] = useState()
+    const [ isCountyDataLoaded, setIsCountyDataLoaded] = useState(false)
 
     useEffect(() => {userPlace && getCountyData()}, [userPlace])
     useEffect(() => {coordinates.longitude && getIncidenceData()}, [coordinates])
     useEffect(() => setErrorMessage(''), [userPlace,coordinates] )
```

Comment:
Why do you need to reset the ErrorMessage?

Reply:
Otherwise its displayed permanently after displayed once. Like that the error disappears after the user submitted a new input.